app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -1,29 +1,30 @@
+import { Button, FullScreen } from '@18f/identity-components';
+import type { MouseEvent, ReactNode, Ref } from 'react';
 import {
   forwardRef,
   useContext,
-  useRef,
-  useState,
-  useMemo,
   useEffect,
   useImperativeHandle,
+  useMemo,
+  useRef,
+  useState,
 } from 'react';
-import { useI18n } from '@18f/identity-react-i18n';
-import { useDidUpdateEffect } from '@18f/identity-react-hooks';
-import { Button, FullScreen } from '@18f/identity-components';
-import type { FullScreenRefHandle } from '@18f/identity-components';
 import type { FocusTrap } from 'focus-trap';
-import type { ReactNode, MouseEvent, Ref } from 'react';
-import AnalyticsContext from '../context/analytics';
-import AcuantContext from '../context/acuant';
-import FailedCaptureAttemptsContext from '../context/failed-capture-attempts';
+import type { FullScreenRefHandle } from '@18f/identity-components';
+import { useDidUpdateEffect } from '@18f/identity-react-hooks';
+import { useI18n } from '@18f/identity-react-i18n';
 import AcuantCamera, { AcuantDocumentType } from './acuant-camera';
+import type { AcuantCaptureFailureError, AcuantSuccessResponse } from './acuant-camera';
+
```

Comment:
I notice the addition of newlines in the imports a few places throughout the changes. Is there any reason for that?

Reply:
Removed those ([commit link](https://github.com/18F/identity-idp/pull/8903/commits/e88675e847bb09d36172a7a6db1ace708e83ee37)).

The reason that happened:
1. I had `sort-imports` a VSCode plugin setup for a different project and forgot that this was the tool sorting any JS/TS imports I touched (I've since fixed this).
2. I was running `yarn lint --fix`
3. The combination (and my unfamiliarity with the codebase) meant I wasn't sure which changes were necessary for linting to pass.